src/main/java/pl/javastart/task/Drink.java

```diff
@@ -0,0 +1,11 @@
+package pl.javastart.task;
+
+public class Drink {
+    String name;
+    float price;
+    boolean alkohol;
+    Ingredient ing1;
+    Ingredient ing2;
+    Ingredient ing3;
+    int drinkVolume;
```

Comment:
Usuń to pole i zamiast tego zdefiniuj zmienną w metodzie main.
Ogólnie pomysł jest dobry, ale:
- trzeba pamiętać, żeby to pole w ogóle ustawić
- jeżeli zmienimy jakiś składnik, to ta wartość może się stać nieaktualna i obiekt będzie nieprawidłowy